app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,193 @@
+      We receive your personal data when you submit it:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>when contacting us for support using the service, to leave feedback or for any other matters</li>
+      <li>as a user of the service</li>
+    </ul>
+
+    <h2 class='govuk-heading-l'>
+      What personal information we will collect
+    </h2>
+
+    <p class="govuk-body">
+      We will collect:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>
+        your name, email and (if applicable) your phone number when you contact us
+      </li>
+      <li>
+        your email address if you include it with feedback to us on the service or agree to be contacted for user research
+        purposes
+      </li>
+    </ul>
+
+    <h2 class='govuk-heading-l'>
+      How we will use your information
+    </h3>
+
+    <p class="govuk-body">
+      If you contact us, DfE and (where applicable) dxw will use your name and contact information for the purpose of
+      responding to you and providing support. We will not share this information with any other parties.
+    </p>
+
+    <h2 class='govuk-heading-l'>
+      Why our use of your personal data is lawful
+    </h2>
+
+    <p class="govuk-body">
+      In order for our use of your personal data to be lawful, we need to meet one (or more) conditions in the data
+      protection legislation. For the purpose of this project, your consent forms the basis of this as stated under GDPR
+      Article 6 (1)(a).
+    </p>
+
+    <h2 class='govuk-heading-l'>
+      Who we will make your personal data available to
+    </h2>
+
+    <p class="govuk-body">
+      We sometimes need to make personal data available to other organisations. These might include contracted partners (who
+      we have employed to process your personal data on our behalf) and/or other organisations (with whom we need to share
+      your personal data for specific purposes).
+    </p>
+
+    <p class="govuk-body">
+      You must not misuse our service by knowingly introducing viruses, trojans, worms, logic bombs or other material
+      that is malicious or technologically harmful. You must not attempt to gain unauthorised access to our service,
+      the server on which our service is stored or any server, computer or database connected to our service. You must
+      not attack our site via a denial-of-service attack or a distributed denial-of service attack. Each of these acts
+      is a criminal offence. We will report any such offence to the relevant law enforcement authorities and co-operate
+      with them to determine your identity.
+    </p>
+
+    <p class="govuk-body">
+      Where we need to share your personal data with others, we ensure that this data sharing complies with data protection
+      legislation. We share specific types of personal data with dxw, who have been contracted by DfE to manage the delivery
+      and hosting of the digital service.
```

Comment:
Not true, dxw are not hosting the digital service.

Reply:
>Will remove this for now